apps/site/src/components/NavBar/NavBar.module.scss

```diff
@@ -3,73 +3,74 @@
 
 .nav {
 	width: 100%;
-	filter: drop-shadow(0px 2.5px 2px rgba(0, 0, 0, 0.4));
-
 	@include utils.media-breakpoint-up(sm) {
-		position: sticky;
+		position: fixed;
+		z-index: 1;
+	}
+
+	@media only screen and (max-width: 991.5px) {
+		background-color: rgba(20, 20, 20, 0.7);
 		z-index: 101;
-		top: 0;
+		position: fixed;
+		text-align: right;
+		backdrop-filter: blur(7px);
 	}
```

Comment:
Suggestion: we prefer to follow a mobile first design approach, following Bootstrap's recommendation. This can be replaced with Bootstrap's own mixin `media-breakpoint-up(lg)` and setting these css properties to be the regular properties and refactoring the other css properties to represent larger viewports.

Reply:
Using the builtin breakpoints seem to mess it up (the background appears for some dimensions with the hamburger menu and not others). The images are from changing line 11 to `media-breakpoint-up(lg)`
![image](https://github.com/user-attachments/assets/302f780f-2eef-41c2-9d91-003770f7754e)
![image](https://github.com/user-attachments/assets/c7a95701-cc1e-472c-ac37-09119c6eb0ab)
![image](https://github.com/user-attachments/assets/58d5ea18-a84c-4066-97e0-67da44d943b4)

Reply:
You have to move the CSS styles around because those styles are only applied when the width of the screen is less than 991.5px. With `media-breakpoint-up(lg)`, the styles will be applied when the width of the screen is larger than the `lg` breakpoint ($>=$ 992px).

Reply:
Mobile first is setting CSS styles to work for mobile displays. Then, for larger displays, with the use of media queries you can override the existing CSS styles or add new ones to work for these larger displays.